src/core/error_helpers.js

```diff
@@ -253,6 +406,8 @@ function helpForMisusedAtTopLevelCode(e, log) {
 
 // Exposing this primarily for unit testing.
 p5.prototype._helpForMisusedAtTopLevelCode = helpForMisusedAtTopLevelCode;
+p5.prototype._validateParameters = validateParameters;
+p5._validateParameters = validateParameters;
```

Comment:
Done reformatting: `p5._validateParameters = function(...`
Linking the function under `prototype` was mainly for bypassing linking issues which occurs during unit tests. I will later double-check and update the unit tests if I can find a good solution.